packages/rancher-logging/generated-changes/overlay/templates/loggings/root/logging.yaml

```diff
@@ -49,6 +64,24 @@ spec:
     image:
       repository: {{ template "system_default_registry" . }}{{ .Values.images.fluentbit.repository }}
       tag: {{ .Values.images.fluentbit.tag }}
+    {{- if or .Values.fluentbit.inputTail.Buffer_Chunk_Size .Values.fluentbit.inputTail.Buffer_Max_Size .Values.fluentbit.inputTail.Mem_Buf_Limit .Values.fluentbit.inputTail.Multiline_Flush .Values.fluentbit.inputTail.Skip_Long_Lines }}
```

Comment:
Discussed with @cbron @brendarearden and concluded that using variables to shorten this if statement would be more confusing due to variables not existing in the other templates. We could use `_helpers.tpl`, but the if statements may not function as intended. Thus, having this long if statement may suffice.